PrioritiesSprint2.md

```diff
@@ -0,0 +1,81 @@
+These are the priorities for Sprint number 2, which runs from March 18, 2024 to April 15, 2024.
+In this sprint, the Octopus build is in CI and the Lobster build is in Test.
+Each priority has a list of subtasks that are to be completed within this sprint.
+
+# Develop Pathfinder Query
+## Pre-development work:
+- Map out upcoming cycles culminating in release to production
+- Initial UI planning and discussions
+- Document the use of TRAPI for Pathfinder queries in the ReasonerAPI github repo.
+- Provide input variables and data examples to the UI team
+## Octopus (CI) work:
+- ARAs implement prototype Pathfinder
+- Evaluate performance and result quality
```

Comment:
Are there more details on how this evaluation will be done?

Reply:
@uhbrar can you comment?